models/source.yml

```diff
@@ -0,0 +1,38 @@
+version: 2
```

Comment:
Once you take a pull from master, you won't have add this file to the PR right?

Reply:
Yes correct, this is already outdate - since i have taken a pull from yours and merged from main. We're good. @evil-in